client/client.go

```diff
@@ -21,7 +21,7 @@ import (
 	"github.com/statechannels/go-nitro/types"
 )
 
-// Client provides the interface for the consuming application
```

Comment:
Nit but should we include this in the PR?

Reply:
No, [6971fb3](https://github.com/statechannels/go-nitro/pull/1083/commits/6971fb3db947ab6f2e660bffdcdbc63db3b4a1a6)